src/contracts/atlas/AtlasVerification.sol

```diff
@@ -548,27 +549,21 @@ contract AtlasVerification is EIP712, DAppIntegration, AtlasConstants {
     function _verifyUser(
         DAppConfig memory dConfig,
         UserOperation calldata userOp,
+        bytes32 userOpHash,
```

Comment:
It seems `userOpHash` isn't used here.

Reply:
I think we should pass it in, and then use it instead of re-computing the userOpHash as a param in the call to `isValidSignatureNow()`, _if and only if_ `allowsTrustedOpHash()` is `false`. If `allowsTrustedOpHash()` is `true` we need to compute the full op hash here as that is how the user will sign the userOp.

Note: this depends on accepting the suggested fix in https://github.com/spearbit-audits/review-fastlane/issues/66 because only then would the userOpHash be computed the same at the top of `_validCalls()` and here when not using alt operation hash.

Reply:
This makes sense, but I'm going to do it in the other PR I think.